src/frequenz/sdk/actor/power_distributing/_battery_status.py

```diff
@@ -352,6 +354,25 @@ def _get_current_status(self) -> Status:
 
         return Status.WORKING
 
+    def _capacity_not_nan(self, msg: BatteryData) -> bool:
+        """Battery capacity should not be guessed.
```

Comment:
What about `Check whether the battery capacity is NaN or not` or something around that

Reply:
Done